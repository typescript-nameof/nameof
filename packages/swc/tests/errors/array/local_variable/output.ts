[
    "x"
];
[
    "x"
];
nameof.array((x)=>[
        nameof.full(x)
    ]);
nameof.array((x)=>[
        nameof.full<typeof x>()
    ]);
