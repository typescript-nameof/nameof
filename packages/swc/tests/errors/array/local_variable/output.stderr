  x Missing a property accessor to the local variable `x`.
   ,-[input.js:3:1]
 2 | nameof.array((x) => x);
 3 | nameof.array((x) => [nameof.full(x)]);
   :                                  ^
 4 | nameof.array((x) => [nameof.full<typeof x>()])
   `----
  x Missing a property accessor to the local variable `x`.
   ,-[input.js:4:1]
 3 | nameof.array((x) => [nameof.full(x)]);
 4 | nameof.array((x) => [nameof.full<typeof x>()])
   :                                  ^^^^^^^^
   `----
